src/test_workflow/integ_test/distribution_tar.py

```diff
@@ -35,3 +36,7 @@ def start_cmd(self) -> str:
             "opensearch-dashboards": "./opensearch-dashboards",
         }
         return start_cmd_map[self.filename]
+
+    def uninstall(self) -> None:
+        logging.info(f"Cleanup {self.work_dir}/* content after the test")
+        subprocess.check_call(f"rm -rf {self.work_dir}/*", shell=True)
```

Comment:
It always worries me to have “rm -rf”, and it won’t work for windows, is there a python method?

Reply:
There is ways to cleanup a folder, but not contents within the folder, unless you list all the files and loop deletion.
Will try researching.